calendar/experiments/calendar/schema/calendar-items.json

```diff
@@ -14,8 +14,67 @@
           "description": { "type": "string", "optional": true },
           "location": { "type": "string", "optional": true },
           "categories": { "type": "array", "items": { "type": "string" }, "optional": true },
+          "priority": { "type": "integer", "optional": true },
+          "privacy": { "type": "string", "optional": true },
+          "status": { "type": "string", "optional": true },
+          "transparency": { "type": "string", "optional": true },
+          "created": { "type": "string", "optional": true },
+          "dtstamp": { "type": "string", "optional": true },
+          "lastModified": { "type": "string", "optional": true },
+          "alarms": { "type": "array", "items": { "$ref": "CalendarItemAlarm" }, "optional": true },
+          "organizer": { "$ref": "CalendarItemAttendee", "optional": true },
+          "attendees": { "type": "array", "items": { "$ref": "CalendarItemAttendee" }, "optional": true },
+          "recurrence": { "$ref": "CalendarItemRecurrence", "optional": true },
+          "occurrenceId": { "type": "string", "optional": true },
+          "occurrenceIndex": { "type": "integer", "optional": true },
           "startDate": { "type": "string", "optional": true },
+          "startTimezone": { "type": "string", "optional": true },
           "endDate": { "type": "string", "optional": true },
+          "endTimezone": { "type": "string", "optional": true },
+          "isAllDayEvent": { "type": "boolean", "optional": true },
+          "entryDate": { "type": "string", "optional": true },
```

Comment:
What we should likely do in the schema is make an item a choice of either an event or a task to make sure the properties are separated.

Reply:
So what you're saying is that you want a number of types:
- CalendarEvent
- CalendarTask
- createEventProperties
- createTaskProperties
- updateEventProperties
- updateTaskProperties

with the existing types then being choice types for the above types? It's doable but it's going to duplicate the already triplicated list of calendar item properties...